### PR TITLE
CY-2601 Fixed agents widget

### PR DESCRIPTION
**Pull request by kubama**

### CODE REVIEW

File: widgets/common/src/NodeFilter.jsx [hunk 1]
```diff
@@ -1,84 +1,116 @@
...
-function toId(resourceName) {
-    return `${resourceName}Id`;
-}
-
 /**
  * NodeFilter  - a component showing dropdowns for filtering blueprints, deployments, nodes and nodes instances.
  * Data (list of blueprints, deployments, nodes and node instances) is dynamically fetched from manager.
  *
+ * @param event
+ * @param field
```
Comment (qooban) [inline widgets/common/src/NodeFilter.jsx:6]: Was that added intentionally? Possibly can be removed?

File: widgets/common/src/NodeFilter.jsx [hunk 2]
```diff
@@ -1,84 +1,116 @@
...
-    allowMultipleNodes,
-    allowMultipleNodeInstances,
-    allowedBlueprints,
-    allowedDeployments,
-    allowedNodes,
-    allowedNodeInstances
-}) {
-    const { useState, useEffect } = React;
-    const { useErrors } = Stage.Hooks;
+export default class NodeFilter extends React.Component {
```
Comment (qooban) [inline widgets/common/src/NodeFilter.jsx:9]: I assume it was too complex to convert to functional component and you reverted it back to class component, right?

**Review by qooban — COMMENTED** [review]
Added some comments.

File: widgets/common/src/NodeFilter.jsx (hunk 2 above)
Comment (kubama) [reply widgets/common/src/NodeFilter.jsx:9]: Yes, this is the case

File: widgets/common/src/NodeFilter.jsx (hunk 1 above)
Comment (kubama) [reply widgets/common/src/NodeFilter.jsx]: Removed

**Review by qooban — APPROVED** [review]

**Pull request merged**